### ISSUE TITLE
Endoscopy - CVAT Integration - Add feature for publishing new model back to CVAT

### ISSUE THREAD

**Issue by tangy5** [Project-MONAI/MONAILabel#963]
The endoscopy app and CVAT integration development is merged for the first version by Sachi and Andres. 

The issue is for continuous development of the Endoscopy-CVAT integration. 

- [x] Currently, the actively trained model in MONAI Label is not publishing back to CVAT. It will continue using pre-trained models. This ticket includes the work on automating the process for transporting fine-tuned model to CVAT. 
- [x] Tool Tracking Segmentation model publishing to CVAT
- [x] Add documentation for CVAT setup and usage with MONAI Label. 
- [x] To watch or add more feature for endoscopy-CVAT usage, issues. Fix or add.
